src/Init/Notation.lean

```diff
@@ -18,25 +18,29 @@ infixl:70 " / "  => Div.div
 infixl:70 " % "  => Mod.mod
 infixl:70 " %ₙ " => ModN.modn
 infixr:80 " ^ "  => Pow.pow
+prefix:100 "-"   => Neg.neg
 
-infix:50 " ≤ "  => HasLessEq.LessEq
+-- declare ASCII alternatives first so that the latter Unicode unexpander wins
```

Comment:
Issue: How to restore the `pp.unicode` behavior? When synthesizing the unexpander, we could reasonably check for Unicode characters in the tokens and emit a check for said option in that case. Notations without ASCII alternatives would be printed as ordinary implications in that case, but perhaps that's actually the more reasonable behavior.

Reply:
I agree.